### PR TITLE
[9.0] [MIG] project_stage_closed

### PR DESCRIPTION
**Pull request by pedrobaeza**
Simple migration of the module for setting the dependency as stated in https://github.com/OCA/project/pull/163#r71844619

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @Tecnativa

File: project_stage_closed/models/task_stage.py [hunk 1]
```diff
@@ -10,21 +11,19 @@ class ProjectTaskType(models.Model):
...
 
     @api.multi
     def _compute_closed_alias_visible(self):
         ir_module = self.env['ir.module.module']
         installed = ir_module.search([('name', '=', 'sale_service'),
                                       ('state', '=', 'installed')])
-        visible = not bool(installed)
-        for o in self:
-            o.closed_alias_visible = visible
+        self.update({'closed_alias_visible': not bool(installed)})
```
Comment (sbidoul) [inline project_stage_closed/models/task_stage.py]: out of curiosity, why is `update` better? the underlying implementation looks very similar.

**Review by dreispt — APPROVED** [review]

**Review by sbidoul — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I have added a test for keeping the perfect coverage.

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Comment by pedrobaeza** [comment]
Can you please merge to continue with the rest of PRs?

**Comment by pedrobaeza** [comment]
Thanks!

**Pull request merged**